### PR TITLE
Edit Permissions help text

### LINKED ISSUE #17739: Grammatical and microcopy fixes for the new Permissions Help sidebar
References to `No access` in the sidebar should be updated to `Block`.

image

image

---

> Users can use the visual query builder to questions based on all tables in this database. A user group must have Unrestricted access for a database if you want to give them access to the SQL/native query editor.

Should read:

> Users can use the visual query builder to ask questions based on all tables in this database. A user group must have Unrestricted access for a database if you want to give them access to the SQL/native query editor.

---

> Restrict user access to specific tables in a database. When you select this option, you’ll be taken to the table-level view of that database to make more granular options.

Should read:

> Restrict user access to specific tables in a database. When you select this option, you’ll be taken to the table-level view of that database to set the access level for each table.

--- 

> Prevent users create new ad hoc queries or questions based on this data, or see this data in the Browse Data screen. But users can still see existing saved questions and charts based on this data in Collections that they have access to.

Should read:

> Prevent users from creating new ad hoc queries or questions based on this data, or from seeing this data in the Browse Data screen. Users with this level of access can still see saved questions and charts based on this data in Collections they have access to.

---

> Ensure users can’t ever see the data from a certain database regardless of their permissions at the collection level. Keep in mind that if the user is part of another group with data access, it will take precedence and their access will not be blocked.

Should read:

> Ensure users can’t ever see the data from this database regardless of their permissions at the Collection level. Keep in mind that if a user belongs to another group that does have data access, that setting will take precedence, and the user's access will not be blocked.

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Edits the copy of permissions sidebar help text according to @mazameli 's suggestions in #17739 
- Closes #17739

### CODE REVIEW

File: frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx [hunk 1]
```diff
@@ -21,32 +21,32 @@ export const DataPermissionsHelp = () => (
       <p>{jt`Users can be members of multiple groups, and are given the ${(
         <strong>{t`most permissive`}</strong>
       )} level of access for a database or table across all the groups they’re a member of.`}</p>
-      <p>{t`Unless a user group has “no access” for a given database or table, they’ll be able to view any saved questions based on that data if they have access to the collection it’s saved in.`}</p>
+      <p>{t`Unless a user group has “block” for a given database or table, they’ll be able to view any saved questions based on that data if they have access to the collection it’s saved in.`}</p>
```
Comment (mazameli) [inline frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx:24]:
```suggestion
      <p>{t`Unless a user group's access for a given database is set to “block," they’ll be able to view any saved question based on that data if they have access to the collection it’s saved in.`}</p>
```

**Review by mazameli — CHANGES_REQUESTED** [review]
Just one small change. Thanks for doing this!

File: frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx]: Done :)

**Review by alxnddr — APPROVED** [review]
Thanks for taking care of it

**Review by mazameli — APPROVED** [review]

**Pull request merged**